deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -31,6 +31,8 @@
 from wazuh_testing import is_udp, is_tcp
 from wazuh_testing.tools.monitoring import wazuh_unpack, Queue
 from wazuh_testing.tools.remoted_sim import Cipher
+from wazuh_testing.tools.utils import retry
+from wazuh_testing.tools.monitoring import Queue
```

Comment:
This import is duplicated

Reply:
https://github.com/wazuh/wazuh-qa/pull/1115/commits/ed3c9388716ce2bfb190a29f8282d6a5bccdcf0e